content/pages/vocational-rehab-and-employment/apply-vre.md

```diff
@@ -1,26 +1,11 @@
 ---
 layout: page-breadcrumbs.html
 title: Apply for VR&amp;E Benefits
-template: 4-action-page
+template: 6-info-page
 ---
 
-<div class="main" role="main" markdown="0">
```

Comment:
@bethpotts you shouldn't need any of this gunk at the top of a template, just start off right with intro text. This is what was making the column so narrow.